docs/readme.md

```diff
@@ -0,0 +1,28 @@
+# Documentation
+
+## Development
+
+For information about the development process, see: [development](./development/development.md)
+
+## Production
+
+For information about the process to deploy on production, see [production](./production/production.md)
+
+## Testing
+
+To fire all unit tests:
+
+Make sure you have setup your work environment (see development).
+
+```
+cd substrate-node
+cargo test
+```
+
+For integration tests, see [integration-tests](../substrate-node/tests/readme.md)
+
+## Workflows
```

Comment:
I don't see clippy integrated, or did I miss it?

Reply:
It's not active for this project